### ISSUE TITLE
TypeError : Invalid URL, when loading directus bundle extension in directus version 10.11.2 on windows

### ISSUE THREAD

**Issue by itszeeshan** [directus/directus#22647]
### Describe the Bug

When we load bundle extension in the latest Directus version, it throws INVALID_URL type error.

### To Reproduce

* Create a new self-hosted project with Postgres as database. https://docs.directus.io/self-hosted/cli.html
* Now have a Directus bundle extension with multiple entries like interfaces, endpoints, displays, hooks extensions.
* The bundle won't load the endpoints and hooks extension but will load the rest of extensions and also will throw this error. This error itself doesn't tell too much of what's actually happening.
![image](https://github.com/directus/directus/assets/35112049/ea75f634-a596-41d1-9b3e-3b9454f21cbf)

### Directus Version

10.11.2

### Hosting Strategy

Self-Hosted (Docker Image)

### Database

PostgreSQL 14.4

**Comment by paescuj** [comment]
Duplicate of #21487. If you have additional information, please comment in that issue report instead. Thank you!

**Comment by itszeeshan** [comment]
But there's still no update on that issue :)

**Comment by paescuj** [comment]
@itszeeshan Yeah, might be helpful if you post any additional information there 👍

**Comment by itszeeshan** [comment]
@paescuj ahh it wasn't something on your end, but ours. Really sorry to bother you on this one :)

**Comment by paescuj** [comment]
> @paescuj ahh it wasn't something on your end, but ours. Really sorry to bother you on this one :)

Great, I'm glad it works for you now 🎉 No worries!
